benchopt/plotting/plot_objective_curve.py

```diff
@@ -167,14 +167,17 @@ def get_solver_color(solver, plotly=True):
     return color
 
 
-def get_solver_marker(solver):
+def get_solver_marker(solver, plotly=True):
     if solver in html_solver_styles and 'marker' in html_solver_styles[solver]:
         return html_solver_styles[solver]['marker']
 
-    markers = {i: i for i, v in enumerate(plt.Line2D.markers)}
+    if plotly:
+        markers = {i: i for i, v in enumerate(plt.Line2D.markers)}
+    else:
+        markers = {i: v for i, v in enumerate(plt.Line2D.markers)}
```

Comment:
Something like this seems a bit easier to follow, with `MARKERS` a constant fo rthe module.

```suggestion
    idx = len(html_solver_styles)
    if plotly:
        marker = idx % len(MARKERS)
    else:
        marker = MARKERS[idx % MARKERS]
```